### PR TITLE
Update the WG Home page

### PR DESCRIPTION
**Pull request by iherman**
Updating the publication snapshot and the home page, with also a reference to the JSON-LD 1.1 EPUB 3 volume.

_This PR should be merged when the Rec-s are published, not before!_

### CODE REVIEW

**Comment by davidlehn** [comment]
Should the epub file be versioned in some way?

To be honest, I'm not even sure how to view a 434KB `.epub` file!  Is this format better for some use case than, say, a single file HTML?

File: PublStatus.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ layout: default
...
-    * Recommendation (REC): June 2020.
-* JSON-LD 1.1 Processing Algorithms and API:
-    * First Public Working Draft (FPWD): September 2018 (✓)
-    * Candidate Recommendation (CR): Q4 2019 (✓)
-    * Recommendation (REC): June 2020.
-* JSON-LD 1.1 Framing:
-    * First Public Working Draft (FPWD): September 2018 (✓)
-    * Candidate Recommendation (CR): Q4 2019 (✓)
-    * Recommendation (REC): June 2020.
+* [JSON-LD 1.1](ttps://www.w3.org/TR/json-ld11-api/):
```
Comment (davidlehn) [inline PublStatus.md:23]:
```suggestion
* [JSON-LD 1.1](https://www.w3.org/TR/json-ld11-api/):
```

Comment (iherman) [reply PublStatus.md]: Oops:-)

**Comment by iherman** [comment]
> Should the epub file be versioned in some way?
>

That makes sense. We can

- set it to jsonld1.1.epub
- set it to jsonld1.1.2020.06.16.epub

although the second is a bit of a mouthful:-(

> To be honest, I'm not even sure how to view a 434KB .epub file! Is this format better for some use case than, say, a single file HTML?

There are a bunch of EPUB readers out there, the quality of some of them is such that they don't do a perfect rendering (in many respects, these W3C documents produce bleeding edge EPUB 3, but an upcoming WG will hopefully make things better). If you are on a Mac, you can load it into the built-in Books app, that would also work on iPhone and Ipad. On Mac or Windows the [Thorium reader](https://www.edrlab.org/software/thorium-reader/) is a pretty good alternative (that is what I use for these).

There is no such thing as a single HTML alternative for all four documents (the core specs plus streaming). But I suspect that if there was one, it would pretty much stretch the browsers. The book also contains a unified table of content for all four documents, it is one single file that you can move around easily among the different machines and can be read off-line and, for very long documents I always find the paginated view better. But that is a matter of taste.

**Comment by gkellogg** [comment]
On Jul 9, 2020, at 9:53 PM, Ivan Herman <notifications@github.com> wrote:
> 
> ﻿
> Should the epub file be versioned in some way?
> 
> That makes sense. We can
> 
> set it to jsonld1.1.epub
> set it to jsonld1.1.2020.06.16.epub
> although the second is a bit of a mouthful:-(
> 

I would have thought it to be unnecessary, as it’s located at a versioned URL.

Gregg

**Comment by iherman** [comment]
> > 
> > That makes sense. We can
> > 
> > set it to jsonld1.1.epub
> > set it to jsonld1.1.2020.06.16.epub
> > although the second is a bit of a mouthful:-(
> > 
> 
> I would have thought it to be unnecessary, as it’s located at a versioned URL.

Not this one. We indeed have EPUB versions for each document whose links and locations are alongside the Rec document itself. But those are an EPUB per document. The one we are talking about in this issue is the EPUB containing all four documents as one volume, and located on the WG's web site.

**Pull request merged**